README.md

```diff
@@ -26,105 +26,106 @@
-- `grunt min`: Same as `grunt`, but outputs a minified style.
-- `grunt user`: Build a userstyle ready to be pasted on userstyles.org.
-- `grunt usermin`: Same as `grunt user`, but outputs a minified style.
-- `grunt usercss`: Build a [Usercss](https://github.com/openstyles/stylus/wiki/Usercss) style, to be used with modern userstyle engines like [stylus](https://github.com/stylus/stylus).
-- `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.
-- `grunt clean`: Reformat `github-dark.css` to conform the style guide.
-- `grunt lint`: Run Stylelint on `github-dark.css`.
-- `grunt authors`: Regenerate the `AUTHORS` file based on git history.
-- `grunt imagemin`: Minify all images present in the `image` directory.
-- `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.
-- `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.
-- `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.
-- `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.
-- `grunt update`: Update and install dependencies.
+:monkey_face: `grunt`: The default task will build a custom theme using a provided `build.json` file. See the [wiki](https://github.com/StylishThemes/GitHub-Dark/wiki/Build) for details.  
+:monkey_face: `grunt min`: Same as `grunt`, but outputs a minified style.  
+:monkey_face: `grunt user`: Build a userstyle ready to be pasted on userstyles.org.  
+:monkey_face: `grunt usermin`: Same as `grunt user`, but outputs a minified style.  
+:monkey_face: `grunt usercss`: Build a [Usercss](https://github.com/openstyles/stylus/wiki/Usercss) style, to be used with modern userstyle engines like [stylus](https://github.com/stylus/stylus).  
+:monkey_face: `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.  
+:monkey_face: `grunt clean`: Reformat `github-dark.css` to conform the style guide.  
+:monkey_face: `grunt lint`: Run Stylelint on `github-dark.css`.  
+:monkey_face: `grunt authors`: Regenerate the `AUTHORS` file based on git history.  
+:monkey_face: `grunt imagemin`: Minify all images present in the `image` directory.  
+:monkey_face: `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.  
+:monkey_face: `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt update`: Update and install dependencies.
```

Comment:
Also excessive here,  I think the old list ist fine in this section

Reply:
mmm blame the monkey...